src/test/regress/expected/failure_multi_row_insert.out

```diff
@@ -134,6 +134,14 @@ SELECT citus.mitmproxy('conn.onQuery(query="^INSERT").cancel(' || :pid || ')');
 
 INSERT INTO reference_table VALUES (1), (2), (3), (4);
 ERROR:  canceling statement due to user request
+SELECT citus.mitmproxy('conn.onQuery(query="^INSERT").cancel(' || :pid || ')');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+INSERT INTO distributed_table VALUES (1,1), (2,2), (3,3), (4,2), (5,2), (6,2), (7,2);
```

Comment:
Maybe  add one  more test with `conn.onQuery(query="^INSERT").after(1).cancel(' || :pid || ')` as well,  which is basically cancel on the second INSERT over  the same connection